spec/components/Web3API_Client.md

```diff
@@ -0,0 +1,157 @@
+# Web3API Client
+
+Web3API client is intended for application developers that can use the client as a single library in their (d)apps to interact with various decentralized services using Web3API's packages.
+
+Currently, Web3API client is available in the following languages:
+ - JavaScript (TypeScript)
+
+## Client interface
+The following snippet contains the interface that client should implement.
+
+```
```

Comment:
Maybe emphasize that those are written in typescript. Some more generic IDL language might be better.